src/main/java/marquez/client/models/FieldType.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package marquez.client.models;
+
+public enum FieldType {
+  ARRAY,
+  BIGINT,
+  BINARY,
+  BOOL,
+  BOOLEAN,
+  BPCHAR,
+  CHAR,
+  CHARACTER,
+  DATE,
+  DATETIME,
+  DECIMAL,
+  DOUBLE,
+  FLOAT,
+  FLOAT4,
+  FLOAT8,
+  GEOMETRY,
+  INT,
+  INT2,
+  INT4,
+  INT8,
+  INTEGER,
+  NCHAR,
+  NUMBER,
+  NUMERIC,
+  NVARCHAR,
+  OBJECT,
+  REAL,
+  SMALLINT,
+  STRING,
+  TEXT,
+  TIME,
+  TIMESTAMP,
+  TIMESTAMPTZ,
+  TIMESTAMP_LTZ,
+  TIMESTAMP_NTZ,
+  TIMESTAMP_TZ,
+  VARBINARY,
+  VARCHAR,
+  VARIANT,
+  VARYING;
+}
```

Comment:
We'll want to make the field type of a dataset a string and allow the marquezAPI to both validate and define allowable types based on source. For more context, checkout [this proposal](https://github.com/MarquezProject/marquez/issues/698).

Reply:
Should this be done once this [issue] (https://github.com/MarquezProject/marquez/issues/708) is resolved first?

Reply:
Good point! Yeah, let's hold off until configurable sources are supported.